cupy/random/distributions.py

```diff
@@ -418,3 +418,32 @@ def vonmises(mu, kappa, size=None, dtype=float):
     """
     rs = generator.get_random_state()
     return rs.vonmises(mu, kappa, size=size, dtype=dtype)
+
+
+def wald(mean, scale, size=None, dtype=float):
+    """Wald distribution.
+
+    Returns an array of samples drawn from the wald distribution. Its
+    probability density function is defined as
+
+    .. math::
+       f(x) = \\sqrt{\\frac{scale}{2\\pi x^3}}\\
+           e^{\\frac{-scale(x-mean)^2}{2mean^2x}},
```

Comment:
Why don't you use `\lambda` and `\mu` to denote scale and mean?

Reply:
I imitated numpy document. but I'll fix it.